cli/functionary/package.py

```diff
@@ -84,3 +86,43 @@ def publish(ctx, path):
         response = post("publish", files={"package_contents": upload_file})
         id = response["id"]
         click.echo(f"Package upload complete\nBuild id: {id}")
+
+
+@package_cmd.command()
+@click.pass_context
+@click.option("--id")
+def buildstatus(ctx, id):
+    """
+    View status for all builds, or build with specific id
+    """
+    if id:
+        results = get(f"builds/{id}")
+        _format_results([results], title=f"Build: {id}")
+    else:
+        results = get("builds").get("results")
+        _format_results(results, title="Build Status")
+
+
+def _format_results(results, title=""):
+    """
+    Helper function to organize table results using Rich
+
+    Args:
+        results: Results to format as a List
+        title: Optional table title as a String
+
+    Returns:
+        None
+    """
+    table = Table(title=title, width=170)
+    console = Console()
+    first_row = True
+    for item in results:
+        list = []
+        for key in item:
```

Comment:
I think it makes sense to just universally exclude the "environment" field.  It's always going to show the same value (the id of the currently set environment), which makes it not entirely useful.  It mostly just ends up taking space.  You can check the `key` and just call `continue` if it's "environment".  Or better yet, maybe create a constant called `EXCLUDED_FIELDS` and have your check be:

```python
if key in EXCLUDED_FIELDS:
    continue
```